### PR TITLE
Feature/phone number

### PR DESCRIPTION
**Pull request by johantjellden**

### CODE REVIEW

**Review by FredrikAreschoug — COMMENTED** [review]
Nice I don't see anything about phone number :slightly_smiling_face:. I like the brake out of `BreachedGuideline`'s but collect them all in one file or one per market.

File: src/main/kotlin/com/hedvig/underwriter/service/UnderwriterImpl.kt [hunk 1]
```diff
@@ -169,6 +63,10 @@ class UnderwriterImpl(
         return validateAndCompleteQuote(quote, underwritingGuidelinesBypassedBy)
     }
 
+    private fun createQuoteData(quoteRequest: QuoteRequest): QuoteData =
+        quoteRequest.incompleteQuoteData?.createQuoteData(quoteRequest)
+            ?: throw IllegalArgumentException("Must provide either house or apartment data")
+
```
Comment (FredrikAreschoug) [inline src/main/kotlin/com/hedvig/underwriter/service/UnderwriterImpl.kt:69]: Nice!

File: src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt [hunk 2]
```diff
@@ -232,26 +234,27 @@ class QuoteServiceImpl(
...
-                    price = completeQuote.price!!,
-                    validTo = completeQuote.validTo
+                    id = quote.id,
+                    price = quote.price!!,
+                    validTo = quote.validTo
                 )
-            }
-        )
+            )
+        }
```
Comment (FredrikAreschoug) [inline src/main/kotlin/com/hedvig/underwriter/service/QuoteServiceImpl.kt:257]: Nice that is way easier to read!

File: src/main/kotlin/com/hedvig/underwriter/service/guidelines/BreachedGuidelinesCodes.kt [hunk 3]
```diff
@@ -1,5 +1,7 @@
 package com.hedvig.underwriter.service.guidelines
 
+typealias BreachedGuidelineCode = String
```
Comment (AlexSJ1990) [inline src/main/kotlin/com/hedvig/underwriter/service/guidelines/BreachedGuidelinesCodes.kt:3]: Nice!

**Review by FredrikAreschoug — DISMISSED** [review]
I don't know if i would keep `BreachedGuidelineCode` hence it's just a string and I think you should deprecate message in the schema as well. :slightly_smiling_face:

**Pull request merged**